library/Freckle/App/Test/Logging.hs

```diff
@@ -1,32 +1,40 @@
 module Freckle.App.Test.Logging
-  ( runCapturedLoggingT
+  ( MonadLogger
+  , LoggingT
+  , runCapturedLoggingT
+  , logLineToText
   ) where
 
 import Freckle.App.Prelude
 
 import Control.Concurrent.Chan
 import Control.Monad.Logger
+import Data.DList (DList)
+import qualified Data.DList as DList
 import UnliftIO.Async
 import UnliftIO.Exception (finally)
 
 -- | Run a 'LoggingT', capturing and returning any logged messages alongside
 --
--- I do not know why 'runChanLoggingT' exists presumably for this purpose, but
--- requires so much more effort to ultimately accomplish.
```

Comment:
Apparently, it's because you should be using `runWriterLoggingT`. And screw you if you need `MonadUnliftIO`. :joy: